src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1364,28 +1364,28 @@ private IOperation FinishVisitingStatement(IOperation originalOperation, IOperat
             return result ?? MakeInvalidOperation(originalOperation.Syntax, originalOperation.Type, ImmutableArray<IOperation>.Empty);
         }
 
-        private void VisitStatements(ArrayBuilder<IOperation> statements)
+        private void VisitStatements(IList<IOperation> statements)
```

Comment:
why this change.  wont' this cause interface dispatches now?

Reply:
Derp. Fixed.